src/Pages/ViewList.js

```diff
@@ -1,26 +1,38 @@
 import React from 'react';
+import { render } from 'react-dom';
 import { useCollection } from 'react-firebase-hooks/firestore';
+import EmptyListPrompt from '../components/EmptyListPrompt';
 import { db } from '../lib/firebase';
 
 const ViewList = ({ token }) => {
   // if (token === null) token = 'default';
 
   const [list, loading, error] = useCollection(db.collection(token));
 
+  const renderList = () => {
```

Comment:
Small piece of feedback: whenever a function returns HTML, we consider it a component; which is effectively what this is. It's just a very simple and helpful component! For that reason though, we should ensure it's capitalized like a component would be. I like you're thinking here! I think we could even expand this to return our Loading component too. That way we just call this component, which crunches all of our render logic for us, and it can be surrounded by elements we always know are there. In this case it's just the h2 element.

Reply:
We could define this component outside of ViewList too. It doesn't have to be exported. We could call it List, and call it within ViewList.